rgw/admin/quota.go

```diff
@@ -0,0 +1,56 @@
+package admin
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+)
+
```

Comment:
Please add a note mentioning [global quotas](https://docs.ceph.com/en/latest/radosgw/admin/#reading-writing-global-quotas) are not supported at the moment